.github/actions/setup-zeebe/action.yml

```diff
@@ -28,22 +28,37 @@ inputs:
     required: false
   secret_vault_address:
     description: 'secret vault url'
-    required: true
+    required: false
   secret_vault_roleId:
     description: 'secret vault roleId'
-    required: true
+    required: false
   secret_vault_secretId:
     description: 'secret valut secret id'
-    required: true
+    required: false
 
 outputs: {}
 
 runs:
   using: composite
   steps:
+    # We will fail the action if it's an internal PR without the secrets. If this occurs it
+    # indicates a configuration failure that needs to be fixed.
+    - name: Secrets check
+      if: |
+        github.event.name == 'pull_request'
+        && github.event.pull_request.head.repo.full_name == 'camunda/zeebe'
+        && (inputs.secret_vault_address == ''
+        || inputs.secret_vault_roleId == ''
+        || inputs.secret_vault_secretId == '')
+      shell: bash
+      run: exit 1
```

Comment:
:upside_down_face: You could put the comment in an "echo" here so that we get the relevant information when it fails